src/components/VmDetail/index.js

```diff
@@ -94,8 +105,11 @@ class VmDetail extends Component {
           <dd>{vm.getIn(['cpu', 'vCPUs'])}</dd>
           <dt><span className='pficon pficon-network' /> Address</dt>
           <dd>{vm.get('fqdn')}</dd>
-          <dt><span className='pficon pficon-screen' /> Console</dt>
+          <dt><span className='pficon pficon-screen' /> Console
+            <a href='#' onClick={this.consoleSettings} className={style['options-btn']}><i className={`glyphicon ${iconClass}`} /></a>
```

Comment:
The up/down arrows are clear from UX perspective, please add text, like

 - ([glyphicon-menu-down] show options)
 - ([glyphicon-menu-up] hide options)